msteams-platform/concepts/deploy-and-publish/appsource-publishing-guidance/prepare-app/office-store-checklist.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Publishing checklist
```

Comment:
Maybe "Publishing package checklist"? Definitely not "manifest checklist" (which is used in various places) as this is much more than just your manifest file.